controllers/instanceController.js

```diff
@@ -2,12 +2,56 @@ const aws = require('aws-sdk');
 const fs = require('fs');
 const path = require('path');
 const cloudformation = new aws.CloudFormation();
+const nanoid = require('nanoid');
 
-const createBaaS = (req, res, next) => {
-  const template = fs.readFileSync(path.resolve(__dirname, '../utils/bastion-development.yaml'), 'utf8');
-  const params = {
-    StackName: 'CreateBaaSInstance',
-    TemplateBody: template,
+const Instance = require('../models/instance');
+const RulePriority = require('../models/listenerRulesPriority');
+
+const newRulePriority = () => {
+  return new Promise((resolve, reject) => {
+    RulePriority.findOne({})
```

Comment:
If you are interested, there is a simpler way of updating an existing record you find. @partemen86 found this when we were doing Mellow:
https://github.com/asungur/mellow/blob/main/server/controllers/cardsController.js#L41